packages/react/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "5.5.7",
   "scripts": {
     "start": "npm-run-all core:build build -p build:ts:watch dev:start",
-    "dev:start": "parcel index.html --cache-dir=./.parcel-cache",
+    "dev:start": "parcel index.html",
```

Comment:
I removed the cache from parcel b/c even with the path fix I couldn't serve the app with out manually destroying the cache and then rebuilding. This seems problematic b/c once the build gets into an error state, the only way to fix it is to stop the app, rebuild the cache and re-serve it. I am open to suggestions for a better DX for this change.